client/src/pages/Home.jsx

```diff
@@ -54,9 +51,43 @@ const Home = () => {
 					)}
 				</div>
 				<div className="h-full bg-secondary w-2/4 sm:flex hidden items-center justify-center px-4">
-					<img src="/landing page image.jpg" className="h-auto rounded-full w-auto object-cover" />
+					<img
+						src="/landing page image.jpg"
+						alt="Landing page illustration"
+						className="h-auto rounded-full w-auto object-cover"
+					/>
 				</div>
 			</div>
+
+			{showAnonymousConfirm && (
+				<div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center">
+					<div className="bg-white dark:bg-gray-800 p-6 rounded-lg shadow-xl max-w-md w-full">
+						<h2 className="text-2xl font-bold mb-4 text-gray-900 dark:text-white">
+							Login Anonymously
+						</h2>
+						<p className="mb-6 text-gray-700 dark:text-gray-300">
+							Are you sure you want to login anonymously? This will create a temporary account.
+						</p>
+						<div className="flex justify-end space-x-4">
+							<button
+								onClick={() => setShowAnonymousConfirm(false)}
+								className="px-4 py-2 text-sm font-medium text-gray-700 bg-gray-200 rounded-md hover:bg-gray-300 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500"
+							>
```

Comment:
@sant0143 we already have a dialog modal you can use, please use that instead

https://github.com/Dun-sin/Whisper/blob/main/client/src/components/Dialog.jsx

Reply:
okay sure will check it